src/kv.rs

```diff
@@ -1,10 +1,19 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+#[cfg(feature = "proptest-derive")]
```

Comment:
I would use `property-testing` rather than `proptest-derive`, since I don't think the latter makes sense without `proptest`